src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/config/ServiceConfigTest.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.config;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.config.ServiceConfigProvider;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@IntegrationTest
+class ServiceConfigTest {
+
+    private static final String TEST_SERVICE_NAME = "bulkscan";
+
+    @Autowired
+    private ServiceConfigProvider serviceConfigProvider;
+
+    @Test
+    public void getCase_should_call_ccd_to_retrieve_the_case_by_ccd_id() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Linguistics Antipattern - The getter 'getCase_should_call_ccd_to_retrieve_the_case_by_ccd_id' should not return void linguistically](https://app.codacy.com/app/HMCTS/bulk-scan-orchestrator/pullRequest?prid=4498197)

Reply:
invalid / copied test name

Reply:
Ohh!